ccmlib/scylla_node.py

```diff
@@ -1215,7 +1219,7 @@ def flush(self, ks=None, table=None):
         super(ScyllaNode, self).flush(ks, table)
         self._wait_no_pending_flushes()
 
-    def get_node_scylla_version(self, scylla_exec_path=None):
+    def run_scylla_executable_with_option(self, option, scylla_exec_path=None):
```

Comment:
maybe make it private? So we won't pollute publi node with too many methods

Reply:
Sure. Fixed